### PR TITLE
Generate PDF

### PR DESCRIPTION
**Pull request by alan-hadyk**
**Important - check and verify new PDF file**

### CODE REVIEW

File: containers/experience/companies/CompanySAP.tsx [hunk 1]
```diff
@@ -3,62 +3,82 @@ import { Company } from "components/organisms/Company/Company";
...
-  title: "Front-End Developer"
-};
+const CompanySAP: React.FC<Pick<ICompanyProps, "format" | "variant">> = ({
+  format,
+  variant
+}) => {
+  const icons = iconsWithLabels
+    .filter(({ iconName }) =>
+      [
+        IconStaticName.Gulp,
```
Comment (angulawk) [inline containers/experience/companies/CompanySAP.tsx:15]: gulp was not presentet before in tech stack for company SAP

File: components/molecules/CompanyDescriptionSection/hooks/useCompanyDescriptionSectionThemeClasses.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  variant = CompanyDescriptionSectionVariant.Blue
+}) => {
+  const companyDescriptionSectionTitleThemeClasses: IThemeClasses = {
+    ...companyDescriptionSectionTitleDefaultThemeClasses,
+    ...mapCompanyDescriptionSectionSizeToTitleStyles[size],
+    color:
+      variant === CompanyDescriptionSectionVariant.Blue
+        ? "text-blue100"
+        : "text-blue600"
+  };
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/CompanyDescriptionSection/hooks/useCompanyDescriptionSectionThemeClasses.ts:23]: Refactored to `mapCompanyDescriptionSectionVariantToTitleStyles`

File: components/molecules/CompanyDescriptionTitle/hooks/useCompanyDescriptionTitleThemeClasses.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  variant = CompanyDescriptionTitleVariant.Light
+}: Pick<
+  ICompanyDescriptionTitleProps,
+  "size" | "variant" | "themeClasses"
+>) => {
+  const companyDescriptionTitleThemeClasses: IThemeClasses = {
+    ...companyDescriptionTitleDefaultThemeClasses,
+    ...mapCompanyDescriptionTitleVariantToStyles[variant],
+    ...mapCompanyDescriptionTitleSizeToStyles[size],
+    ...themeClasses
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/CompanyDescriptionTitle/hooks/useCompanyDescriptionTitleThemeClasses.ts:25]: Refactored:

image

File: components/molecules/Console/screens/ConsolePdf.tsx [hunk 4]
```diff
@@ -6,16 +6,18 @@ import { CornerVariant } from "components/atoms/Corner/@types/Corner";
...
-  consolePdfDefaultThemeClasses,
-  consolePdfInnerContainerDefaultThemeClasses
-} from "components/molecules/Console/styles";
 
 const ConsolePdf: React.FC<Pick<IConsoleTextProps, "children">> = ({
   children
 }) => (
-  <LayoutContainer themeClasses={consolePdfDefaultThemeClasses}>
-    <LayoutContainer themeClasses={consolePdfInnerContainerDefaultThemeClasses}>
+  <LayoutContainer>
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/Console/screens/ConsolePdf.tsx:13]: Nope - that `paddingTop` was a mistake, so I had to remove it.

File: components/molecules/LanguagesGroup/styles.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { IThemeClasses } from "types/theme";
+
+export const languagesGroupDefaultThemeClasses: IThemeClasses = {
+  display: "flex",
+  flexFlow: "flex-row-wrap",
+  gap: "gap-8",
+  width: "w-full"
+};
+
+export const languagesGroupLanguageWrapperDefaultThemeClasses: IThemeClasses = {
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/LanguagesGroup/styles.ts:10]: Refactored: 

image

File: containers/experience/companies/CompanyDiH.tsx [hunk 6]
```diff
@@ -3,53 +3,65 @@ import { Company } from "components/organisms/Company/Company";
...
-  title: "Front-End Developer"
-};
+const CompanyDiH: React.FC<Pick<ICompanyProps, "format" | "variant">> = ({
+  format,
+  variant
+}) => {
+  const icons = iconsWithLabels
+    .filter(({ iconName }) =>
+      [
+        IconStaticName.Grunt,
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply containers/experience/companies/CompanyDiH.tsx:15]: Yes - it was added on purpose.

File: components/molecules/VerticalIconsWithLabels/VerticalIconsWithLabels.tsx [hunk 7]
```diff
@@ -48,22 +52,31 @@ const VerticalIconsWithLabels: React.FC<IVerticalIconsWithLabelsProps> = ({
...
       >
         {iconsWithLabels.map(
-          ({ label }: IIconWithLabelProps): JSX.Element => (
+          ({ href, isExternal, label }: IIconWithLabelProps): JSX.Element => (
             <LayoutContainer
               key={label}
-              themeClasses={
-                verticalIconsWithLabelsLabelContainerDefaultThemeClasses
-              }
+              themeClasses={verticalIconsWithLabelsLabelContainerThemeClasses}
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/VerticalIconsWithLabels/VerticalIconsWithLabels.tsx:61]: Refactored styles in this component:

image

File: components/molecules/VerticalIconsWithLabels/hooks/useVerticalIconsWithLabelsThemeClasses.ts [hunk 8]
```diff
@@ -1,28 +1,77 @@
...
+
+  // Icon
   const verticalIconsWithLabelsIconThemeClasses: IThemeClasses = {
+    ...mapSizeToIconStyles[size],
+    height: mapSizeToIconHeight[size]
+  };
+
+  // Labels wrapper
+  const verticalIconsWithLabelsLabelsWrapperThemeClasses: IThemeClasses = {
+    ...verticalIconsWithLabelsLabelsWrapperDefaultThemeClasses,
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply components/molecules/VerticalIconsWithLabels/hooks/useVerticalIconsWithLabelsThemeClasses.ts:50]: https://github.com/alan-hadyk/alan-hadyk.github.io/pull/313#discussion_r1175700636

File: containers/experience/companies/CompanySAP.tsx (hunk 1 above)
Comment (alan-hadyk) [reply containers/experience/companies/CompanySAP.tsx:15]: Added on purpose

**Review by angulawk — APPROVED** [review]

**Pull request merged**